mir-module/src/main/java/org/mycore/mir/migration/MIRMigration202006Utils.java

```diff
@@ -39,13 +44,58 @@
 import org.mycore.mir.editor.MIRPostProcessor;
 import org.mycore.mir.editor.MIRUnescapeResolver;
 import org.mycore.mods.MCRMODSWrapper;
+import org.xml.sax.SAXException;
 
 @MCRCommandGroup(
     name = "MIR migration 2020.06")
 public class MIRMigration202006Utils {
 
     public static final Logger LOGGER = LogManager.getLogger();
 
+    private static final MCRCategoryID CONTENT_TYPE_ID = MCRCategoryID.fromString("derivate_types:content");
+
+    @MCRCommand(syntax = "migrate derivate display to category {0}",
+        help = "link derivates with @display=false to the given category {0}")
+    public static List<String> migrateDerivateDisplay(String targetCategory) {
+        MCRCategoryID categoryID = MCRCategoryID.fromString(targetCategory);
+        if (!MCRCategoryDAOFactory.getInstance().exist(categoryID)) {
+            throw new MCRException("Category " + categoryID + " does not exist!");
+        }
+        final MCRXMLMetadataManager mcrxmlMetadataManager = MCRXMLMetadataManager.instance();
+        final List<String> derivates = mcrxmlMetadataManager.listIDsOfType("derivate");
+        return derivates.stream()
+            .map(MCRObjectID::getInstance)
+            .map(mcrid -> {
```

Comment:
This always looks ugly, maybe its worth to include apache-commons-lang3. You could then do this:
`Functions.apply(mcrxmlMetadataManager::retrieveXML, mcrid)`

There are other cases alone in this file where we could use this.

Reply:
http://commons.apache.org/proper/commons-lang/javadocs/api-3.9/index.html?org/apache/commons/lang3/Functions.html